pagerduty/extension_schema.go

```diff
@@ -0,0 +1,60 @@
+package pagerduty
+
+import "fmt"
+
+// ExtensionSchemaService handles the communication with extension schemas related methods
+// of the PagerDuty API.
+type ExtensionSchemaService service
+
+// ExtensionSchema represents an extension schema.
+type ExtensionSchema struct {
+	ExtensionSchema *ExtensionSchema `json:"extension_schema,omitempty"`
```

Comment:
Same as above, why is there a pointer to the struct being defined? Can an extension_schema link to another extension_schema?